formats/financial_releases_campaign/publisher/details.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+   "required": [
+     "image",
+     "body"
+   ],
+  "properties": {
+     "image": {
+       "type": "object",
+       "additionalProperties": false,
+       "properties": {
+         "url": {
+           "type": "string",
+           "format": "uri"
+         },
+         "alt_text": {
+           "type": "string"
+         }
+       }
+     },
+    "header_legal_disclaimer": {
+      "type": "string"
+    },
+    "title": {
```

Comment:
Sorry, I must have missed it accidentally. There's now a new commit removing this and the other title that you've pointed out.